libs/blocks/library-config/lists/placeholders.js

```diff
@@ -0,0 +1,28 @@
+import { createTag } from '../../../utils/utils.js';
+import createCopy from '../library-utils.js';
+
+async function fetchPlaceholders(path) {
+  const resp = await fetch(path);
+  if (!resp.ok) return;
+  const json = await resp.json();
+  if (!json.data) return;
+  return json.data;
```

Comment:
total edge case, but if returning `undefined` then L14 will fail.
```suggestion
  if (!resp.ok) return [];
  const json = await resp.json();
  return json.data || [];
```